options.mk

```diff
@@ -5,6 +5,16 @@ ifeq ($(MEASURED_BOOT),1)
   CFLAGS+=-D"WOLFBOOT_MEASURED_PCR_A=$(MEASURED_PCR_A)"
 endif
 
+## use tpm to store true pubkey
+ifeq ($(WOLFBOOT_TPM_KEYSTORE),1)
```

Comment:
Can you create an example .config that has `WOLFBOOT_TPM_KEYSTORE`? For testing?

Reply:
+1 on this, ideally if it runs on STM32F407 so I can integrate with automated TPM tests on Jenkins